### PR TITLE
#683 Update to Bootstrap 4.2.1

### PR DESCRIPTION
**Pull request by ConnorSheremeta**
Upgraded to 4.2.1. Noticed text-decoration had a new 'none' utility.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -48,7 +48,7 @@ gem 'omniauth'
 gem 'omniauth-saml'
 
 # Authorization
-gem 'pundit'
+gem 'pundit', '1.1.0'
```
Comment (murny) [inline Gemfile]: Locking this version to 1.1.0 (last versio before 2.0.0).

As 2.0.0 has breaking changes.

Created an issue for this here to revisit in the future: https://github.com/ualbertalib/jupiter/issues/965

**Comment by murny** [comment]
Haha yep! I mentioned the changelog entry and explained it to Conner at the end of the day yesterday so haven't had enough time to add the commit. But we will definitely add one for this 👍

**Comment by ConnorSheremeta** [comment]
Done! In an embarrassing 3 commits haha, I'll ensure that I squash them all!

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,9 @@ and releases in Jupiter project adheres to [Semantic Versioning](http://semver.o
 ### Fixed
 - workarounds for Datacite EZ API for tests [PR#945](https://github.com/ualbertalib/jupiter/pull/945)
 
+### Changed
+- Update to Bootstrap 4.2.1 [PR#964](https://github.com/ualbertalib/jupiter/pull/964)
```
Comment (pgwillia) [inline CHANGELOG.md]:
```suggestion
- Update to Bootstrap 4.2.1 [#683](https://github.com/ualbertalib/jupiter/issues/683)
```
Typically the issue will have more information about what, why and how for the change than the pull request.  I think it makes more sense to link to the original issue. Would you agree?

**Comment by weiweishi** [comment]
you-did-good-job-meme
Now @ConnorSheremeta you will have the honor to merge the PR, once the remaining conflict is resolved. I think there were some gem upgrades after your PR.

**Pull request merged**